params/protocol_params.go

```diff
@@ -20,13 +20,16 @@ import "math/big"
 
 var (
 	TargetGasLimit = GenesisGasLimit // The artificial target
-)
 
-const (
+	// the followings are modified by Quorum and purposely moved from `const` section
+	// so to faclitate the execution of Ethereum Tests
+	// @see tests/init_test.go#TestMain
 	GasLimitBoundDivisor uint64 = 4096      // The bound divisor of the gas limit, used in update calculations.
 	MinGasLimit          uint64 = 700000000 // Minimum the gas limit may ever be.
 	GenesisGasLimit      uint64 = 800000000 // Gas limit of the Genesis block.
```

Comment:
I prefer to keep the params constant and define Quorum specific constant params.